MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -464,6 +466,20 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func fetchPoints(with callbackId: String) {
+         if isUserAllowedPermission(customPermissionType: .points, callbackId: callbackId) {
+            hostAppMessageDelegate?.getPoints { (result) in
+                switch result {
+                case .success(let response):
+                    guard let encodedPoints = ResponseEncoder.encode(data: response) else { return }
```

Comment:
You need call `self.executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: ERROR_DETAILS_HERE)` before return